Sources/Apodini/Semantic Model Builder/GraphQL/GraphQLInterfaceExporter.swift

```
@@ -0,0 +1,150 @@
+                                            ></script>
+                                            <script>
+                                              const graphQLFetcher = graphQLParams =>
+                                                fetch('/graphql', {
+                                                  method: 'post',
+                                                  headers: { 'Content-Type': 'application/json' },
+                                                  body: JSON.stringify(graphQLParams),
+                                                })
+                                                  .then(response => response.json())
+                                                  .catch(() => response.text());
+                                              ReactDOM.render(
+                                                React.createElement(GraphiQL, { fetcher: graphQLFetcher }),
+                                                document.getElementById('graphiql'),
+                                              );
+                                            </script>
+                                          </body>
+                                        </html>
+                                        """
+
+        return Vapor.Response(status: .ok, headers: ["Content-Type": "text/html"], body: html)
+    }
+
+    private func graphqlServer(_ req: Vapor.Request) throws -> EventLoopFuture<String> {
+        guard let body = req.body.string else {
+            throw NetworkError.noBody
+        }
+
+        // Create Swift dict
+        var query = "{}"
+        let data = body.data(using: .utf8)!
+        do {
+            if let jsonArray = try JSONSerialization.jsonObject(with: data, options: .allowFragments) as? [String: AnyObject] {
+                if let q = jsonArray["query"] as? String {
+                    query = q
+                } else {
+                    print("jsonArray doesn't have query parameter")
```

If the query isn't present we should throw an error